pubmed_dl/main.py

```diff
@@ -1,18 +1,23 @@
-#!/usr/bin/env python
 import json
-from pubmed_dl.ncbi import get_list_pmid, uids_to_docs
+import requests
+import os
+import sys
+from ncbi import uids_to_docs , get_list_pmid
 
 def main():
     ab = open("test.json", 'w')
-    start = input("Enter start date (YYYY/MM/DD): ")
-    end = input("Enter end date (YYYY/MM/DD): ")
+    #start = input("Enter start date (YYYY/MM/DD): ")
+    #end = input("Enter end date (YYYY/MM/DD): ")
+    start = "2020/12/04"
+    end = "2020/12/07"
     val = get_list_pmid(start,end)
+    val[-1] = "bruh"

```

Comment:
I would think almost all of this should be reverted before committing, looks like it's related to your testing. In the near-future we should move testing _outside_ the actual code.